.stickler.yml

```diff
@@ -0,0 +1,3 @@
+linters:
+  stylelint:
+  csslint:
```

Comment:
#### `REQUIRED`

Add `.stickler.yml` file to the root of the project. Also let's use [this file](https://github.com/microverseinc/linters-config/blob/master/css/.stickler.yml). :point_down: 

``` yml
linters:
  stylelint:
    config: 'stylelint.config.js'
```
- Also, add the `stylelint.config.js` that was referenced above, this can be found [here](https://github.com/microverseinc/linters-config/blob/master/css/stylelint.config.js)

If you are having an issue with setting up linter, read the `Troubleshooting` section of [this repo](https://github.com/microverseinc/linters-config/tree/master/css).

Reply:
> Add .stickler.yml file to the root of the project

Do u mean that I have to moove those file to the master Branch ?

Because I've for the moment all of them in the 'dev' branch